src/components/loader-spinner/loader-spinner.spec.tsx

```diff
@@ -56,6 +56,20 @@ describe("LoaderSpinner", () => {
   });
 
   describe("when custom props are passed", () => {
+    it("when the 'spinnerLabel' prop is passed a custom string value it overrides the default visible label", () => {
```

Comment:
**nitpick(non-blocking):** Test descriptions should make sense as a sentence. 

Something like:
```suggestion
    it("should allow a custom string value to be passed to the 'spinnerLabel' prop to override the default visible label", () => {
```
Feel free to make it something better than my suggestion if you want to 😄